game-core/src/main/java/games/strategy/triplea/delegate/battle/firinggroups/Aa.java

```diff
@@ -0,0 +1,62 @@
+package games.strategy.triplea.delegate.battle.firinggroups;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.GamePlayer;
+import games.strategy.engine.data.Unit;
+import games.strategy.engine.data.UnitType;
+import games.strategy.triplea.attachments.TechAbilityAttachment;
+import games.strategy.triplea.attachments.UnitAttachment;
+import games.strategy.triplea.delegate.Matches;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
+import lombok.Builder;
+import lombok.NonNull;
+import org.triplea.java.collections.CollectionUtils;
+
+@Builder
+public class Aa {
```

Comment:
Sure.  I thought since it was inside of the "firinggroups" package, that would indicate that it was a firing group.  Calling it `AaFiringGroup` felt a little redundant.  But I can change its name.

Reply:
Since it's public it can be used outside of the package, makes it less clear at the point of use. Though, I'd general agree that reducing the redundant nouns/specifiers is a good thing though.